hugo/content/survey/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "The Accelerate State of DevOps Survey"
+date: 2023-04-12T15:12:48Z
+draft: false
+---
+
+# The Accelerate State of DevOps Survey
+Over the past eight years, more than 33,000 professionals around the world have taken part in the Accelerate State of DevOps survey, making it the largest and longest-running research of its kind. Year after year, [Accelerate State of DevOps Reports](/publications/) provide data-driven industry insights that examine the capabilities and practices that drive software delivery, as well as operational and organizational performance.
+
+In addition to contributing to our collective understanding of software delivery, the survey offers practitioners a time to reflect on challenges and opportunities within their own teams.
+
+### Participate in the Accelerate State of DevOps Survey
+The 2023 survey will be launching soon! Check back here for more information, or join [dora.community](https://dora.community) for ongoing updates.
```

Comment:
how would you feel about this copy instead?

`[join the DORA Community Google Group](https://dora.community) for ongoing updates.`

From this page you are at least 2 clicks away from joining so providing a hint on the next link you need to follow might be helpful.

Reply:
Yes, makes sense! Updated.